platform/security/expansion/security-expansion-impl/src/main/java/ddf/security/expansion/impl/AbstractExpansion.java

```diff
@@ -471,7 +471,7 @@ protected void loadConfiguration(String filename) {
                 // absolute path
                 file = new File(filename);
             }
-            br = new BufferedReader(new FileReader(file));
+            br = new BufferedReader(new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8.name()));
```

Comment:
Added try with resources here. This originally addressed the encoding issue, but introduced some leaks.